mne/preprocessing/ica.py

```diff
@@ -237,6 +286,15 @@ class ICA(ContainsMixin):
     .. [4] Ablin, P., Cardoso, J.F., Gramfort, A., 2017. Faster Independent
            Component Analysis by preconditioning with Hessian approximations.
            arXiv:1706.08171
+
+    .. [5] Artoni, F., Delorme, A., und Makeig, S, 2018. Applying Dimension
+           Reduction to EEG Data by Principal Component Analysis Reduces the
+           Quality of Its Subsequent Independent Component Decomposition.
+           NeuroImage 175, pp.176–187.
+           https://sccn.ucsd.edu/%7Earno/mypapers/Artoni2018.pdf
+
+     .. [6] https://sccn.ucsd.edu/wiki/Chapter_09:_Decomposing_Data_Using_ICA#Issue:_ICA_returns_near-identical_components_with_opposite_polarities  # noqa
```

Comment:
URLs should not go in References but be standard RST links like:
```
... see the `decomp site <ucsd_site_>`_.

.. _ucsd_site: https://blahblah
```
See for example how we do it here:

https://github.com/mne-tools/mne-python/pull/5692/files#diff-2b866ae21caee8720a60af18064ddf89R9